src/config.rs

```diff
@@ -505,6 +505,20 @@ pub enum Command {
         soft: bool,
     },
 
+    /// Deduplicate files.
+    ///
+    /// The list of groups earlier produced by `fclones group` should be submitted
+    /// on the standard input. Only the default text format is supported.
+    ///
+    /// Can not cross filesystem boundaries, not all filesystems support deduplication.
+    /// The result is not visible to userland applications, so repreated runs
+    /// will find the same files again.
+    Dedupe {
+        // TODO! --priority and --rf-over and make little sense
```

Comment:
Well, reflink might not preserve some metadata of the files, so I think these might be to some degree useful.
I'd leave them just to allow selecting which files are touched.

Reply:
Ok, in this case `--priority` makes sense. The replication factor flag however needs to change or come with a `--first-dedupe-run-i-promise` because no userland app can easily determine if a file is deduplicated on the block level.